index.bs

```diff
@@ -3627,7 +3627,7 @@ as well. This allows authenticators without a font rendering engine to be used a
 :: `txAuthGeneric`
 
 : Client extension input
-:: A CBOR map defined as follows:
+:: A JSON object defined as follows:
     ```
     txAuthGenericArg = {
                            contentType: text,   ; MIME-Type of the content, e.g. "image/png"
```

Comment:
As requested during the meeting:

Rather than a JSON object, we might want a Javascript object as JSON cannot include raw bytes. With a Javascript object, the types of the fields probably want to be DOMString or USVString, and ArrayBuffer.